build_library/grub_install.sh

```diff
@@ -144,7 +144,7 @@ info "Generating ${GRUB_DIR}/load.cfg"
 # The existing $root value is given as a hint so it is searched first.
```

Comment:
This comment became outdated with your change.

Reply:
thanks, updated the comment